apple/internal/partials/framework_import.bzl

```diff
@@ -165,6 +165,7 @@ def _framework_import_partial_impl(ctx, targets, targets_to_avoid):
             outputs = [framework_zip],
             arguments = [args],
             mnemonic = "ImportedDynamicFrameworkProcessor",
+            execution_requirements = {"no-sandbox": "1"},
```

Comment:
This _works_, but we have it conditionally add in `if provisioning_profile:` instead. that way we only jump out of the sandbox when we are signing.

Reply:
Hrm interesting. I wonder what the ramifications are, if any, around conditionally executing in the sandbox. I've never seen that pattern before.

Reply:
Updated with that behavior